memcache.py

```diff
@@ -243,9 +244,9 @@ def __init__(self, servers, debug=0, pickleProtocol=0,
     def _encode_key(self, key):
         if isinstance(key, tuple):
             if isinstance(key[1], six.text_type):
-                return (key[0], key[1].encode('utf8'))
+                return (key[0], key[1].encode('utf-8'))
```

Comment:
Please don't make unrelated cleanups like that in a bug fix branch as it makes review more difficult, do it in a separate PR. (You've accidentally merged in the `touch()` fixes on this branch as well.